DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: calendar
-Title: Create, Read, Write, and Work with 'iCalander' Files, Calendars and
+Title: Create, Read, Write, and Work with 'iCalendar' Files, Calendars and
```

Comment:
Woa typo in the title? Not sure how that slipped the net..

Reply:
It happens - I misspelled my own name once! 🤷‍♀️